policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_min_head_spouse_earned.py

```diff
@@ -0,0 +1,65 @@
+from policyengine_us.model_api import *
+
+
+class hi_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.cdcc
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        # earned minimum income if is disabled/full-time student
+        head_eligible = head & (
+            person("is_disabled", period)
+            | person("is_full_time_student", period)
+        )
+        spouse_eligible = spouse & (
+            person("is_disabled", period)
+            | person("is_full_time_student", period)
+        )
+        # 2400 --> one dependent, 4800 --> more than one dependent
+        qualified_children = tax_unit("count_cdcc_eligible", period)
+        income = person("earned_income", period)
+        head_income = where(
```

Comment:
separate each where and max statement into one line - for visibility

Reply:
same for spouse income

Reply:
try not to use any embedded functions at all

Reply:
still a lot of embeddings - each max_ or where statement should have its own line